tasks/hub-compiled.yml

```
@@ -0,0 +1,45 @@
+---
+
+# hub command
+
+- name: check hub installed
+  shell: which hub
+  register: which_hub
+  failed_when: no
+  changed_when: which_hub.rc != 0
+
+- name: check hub version
+  when: which_hub is not changed
+  shell: hub --version
+  register: hub_ver
+  failed_when: no
+  changed_when: |
+    ('hub version ' + hub_version) != hub_ver.stdout_lines[1]
+
+- name: install or update hub
+  when: which_hub is changed or hub_ver is changed
+  vars:
+    working_dir: /tmp
```

TODO: save in `/opt`

```suggestion
    working_dir: /opt
```